File: DanceEvent/src/Entity/Dj.java
package Entity;

import java.io.Serializable;

public class Dj extends BaseEntity implements Serializable {

    private String name;
    private int age;
    private String style;

    public Dj() {
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }
    public void setAge(int age) {
        this.age = age;
    }

    public String getString() {
        return style;
    }
    public void setString(String string) {
        this.style = string;
    }
}

File: DanceEvent/DanceEvent.Application/Dj.cs
using System;

namespace DanceEvent.Application
{
    public class Dj
    {
        public Dj(long id, string name, int age, string style)
        {
            Id = id;
            Name = name;
            Age = age;
            Style = style;
        }

        public long Id { get; set; }
        public string Name { get; set; }
        public int Age { get; set; }
        public string Style { get; set; }
    }
}
